packages/sampler/stories/default/Drawing.js

```diff
@@ -0,0 +1,31 @@
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import {boolean, range, select} from '@enact/storybook-utils/addons/controls';
+
+import Drawing from '../../../ui/Drawing';
+
+const DrawingConfig = mergeComponentMetadata('Drawing', Drawing);
+
+export default {
+	title: 'UI/Drawing',
```

Comment:
Please sort.

Reply:
this is the order used in every sample. we should leave this as it is now